extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/config/MongoEntityConfigProvider.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.mongodb.panache.config;
+
+import static org.eclipse.microprofile.config.ConfigProvider.getConfig;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+
+import io.quarkus.mongodb.panache.MongoEntity;
+
+public class MongoEntityConfigProvider {
```

Comment:
It's not really a config provider but a config helper.

Maybe `MongoConfigHelper` ? Or `MongoDatabaseNameHelper` ?